releasenotes/notes/fix-docs-starlette-env-var-9901a95294b40a84.yaml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
We typically don't do changelog entries for documentation updates - just major functionality changes or bugs, etc 🙂